.github/workflows/pr-verify.yml

```diff
@@ -3,6 +3,25 @@ name: PR verify
 on: pull_request
 
 jobs:
+  formatting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK
+      uses: actions/setup-java@v3
+      with:
+        distribution: 'temurin'
+        java-version: 17
+        cache: 'maven'
+    - name: Check formatting
+      run: mvn -B formatter:validate impsort:check xml-format:xml-check
+    - name: Cancel workflow on failure
+      uses: vishnudxb/cancel-workflow@v1.2
+      if: failure()
+      with:
+        repo: eclipse/rdf4j
+        workflow_id: ${{ github.run_id }}
+        access_token: ${{ github.token }}
```

Comment:
I think that if we make this its own job, we can get rid of this cancel step.

Reply:
Yes, but don't we want to cancel the other jobs because they will need to run with the next push that has a formatting fix? So cancelling now saves some energy?

Reply:
Ah good point. I had it in my head that this would only cancel the current job - but if it also kills the parallel jobs then it's definitely worth leaving in.